offload/cmake/caches/AMDGPUBot.cmake

```diff
@@ -0,0 +1,13 @@
+set(LLVM_ENABLE_PROJECTS "clang;lld" CACHE STRING "")
+set(LLVM_ENABLE_RUNTIMES "compiler-rt;openmp;offload" CACHE STRING "")
+set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR ON CACHE BOOL "")
+set(LLVM_ENABLE_ASSERTIONS ON CACHE BOOL "")
+set(LLVM_LIT_ARGS "-v --show-unsupported --timeout 100 --show-xfail -j 32" CACHE STRING "")
+set(LLVM_TARGETS_TO_BUILD "X86;AMDGPU" CACHE STRING "")
+
+set(CLANG_DEFAULT_LINKER "lld" CACHE STRING "")
+
+set(CMAKE_INSTALL_PREFIX /tmp/llvm.install.jpbot CACHE STRING "")
+set(CMAKE_BUILD_TYPE Release CACHE STRING "")
+set(CMAKE_C_COMPILER_LAUNCHER ccache CACHE STRING "")
+set(CMAKE_CXX_COMPILER_LAUNCHER ccache CACHE STRING "")
```

Comment:
Can you add a brief doc string as well?

Reply:
Sorry, not sure what to document here other than state "kind of obvious" things.

Reply:
Yeah, that's a fair point. I wish CMake could make it an optional argument. `""` really doesn't look nice.